pipeline_ukbiobank/cli/compute_stats.py

```diff
@@ -231,17 +256,32 @@ def compare_sex(df, path):
 
     # Compute T-test
     results = scipy.stats.ttest_ind(df[df['Sex'] == 0]['T1w_CSA'], df[df['Sex'] == 1]['T1w_CSA'])
+    df.loc[df['Sex'] == 0, 'Sex'] = 'F'
+    df.loc[df['Sex'] == 1, 'Sex'] = 'M'
     # Violin plot
     plt.figure()
+    fig, ax = plt.subplots()
     plt.title("Violin plot of CSA and sex")
     sns.violinplot(y='T1w_CSA', x='Sex', data=df, palette='flare')
-    fname_fig = os.path.join(path,'violin_plot.png')
+    # Add mean CSA and std for female
+    textstr_F = '\n'.join((
+                           r'$\mu=%.2f$' % (mean_csa_F, ),
+                           r'$\sigma=%.2f$' % (std_csa_F, )))
+    ax.text(0.05, 0.92, textstr_F, transform=ax.transAxes, fontsize=14,
+            verticalalignment='top', horizontalalignment='left')
+    # Add mean CSA and std for female
+    textstr_M = '\n'.join((r'$\mu=%.2f$' % (mean_csa_M, ),
+                           r'$\sigma=%.2f$' % (std_csa_M, )))
+    ax.text(0.82, 0.92, textstr_M, transform=ax.transAxes, fontsize=14,
+            verticalalignment='top', horizontalalignment='left')
+    plt.ylabel('CSA (mm$^2$)')
+    fname_fig = os.path.join(path, 'violin_plot.png')
     plt.savefig(fname_fig)
 
     # Write results
     logger.info('Created: ' + fname_fig)
-    logger.info('Mean CSA value for female : {:.4} mm^2, std is {:.4}'.format(mean_csa_F, std_csa_F))
-    logger.info('Mean CSA value for male : {:.4} mm^2, std is {:.4}'.format(mean_csa_M, std_csa_M))
+    logger.info('Mean CSA value for female : {:.4} mm\u00b2, std is {:.4}'.format(mean_csa_F, std_csa_F))
```

Comment:
same comment about the weird ASCII character, but i guess you have a good reason for doing so